maxim/scribe.py

```
@@ -15,52 +15,114 @@ class Scribe:
     """
 
     def __init__(self, name):
```

Ruff (0.11.9)

Missing return type annotation for special method `__init__`

Add return type annotation: `None`

(ANN204)